l10n_ch_dta/__openerp__.py

```diff
@@ -30,9 +30,8 @@
              'account_payment_order',
              'document'],
  'data': ['data/account_payment_method.xml',
-          ],
- 'demo': ['../account/test/account_minimal_test.xml',
```

Comment:
This change is the wrong way to go. This was fixed in https://github.com/OCA/l10n-switzerland/pull/299/commits/cd2258ad176bd6554dc3ad1db677d9ae6a8649b2